README.md

```diff
@@ -1,3 +1,10 @@
 # GlobalX.ChatBots.Core
 
+<p align="center">
+    <a href="https://www.nuget.org/packages/GlobalX.ChatBots.Core/"><img src="https://flat.badgen.net/nuget/v/globalx.chatbots.core" alt="GlobalX.ChatBots.Core nuget package" /></a>
+    <a href="https://travis-ci.org/GlobalX/GlobalX.ChatBots.Core"><img src="https://flat.badgen.net/travis/GlobalX/GlobalX.ChatBots.Core" alt="Pockybot on Travis CI" /></a>
+    <img src="https://flat.badgen.net/github/commits/globalx/globalx.chatbots.core" alt="commits" />
```

Comment:
Is total number of commits really a useful metric to display?

Reply:
Especially given this package probably won't change that much. Still, I don't think it hurts to have it here.